src/aatoolbox/datasources/datasource.py

```diff
@@ -67,7 +67,7 @@ def _config_attribute_name_validator(
             return getattr(country_config, config_datasource_name)
```

Comment:
Oops while you're here, can you make this a @staticmethod, thanks so much (my bad)